### PR TITLE
Prevent MissingWebViewPackageException

### PR DESCRIPTION
**Pull request by capezzbr**
# What does this PR do

When `com.google.android.webview` is not found by the OS, `Fatal Exception: android.util.AndroidRuntimeException: android.webkit.WebViewFactory$MissingWebViewPackageException: Failed to load WebView provider: No WebView installed` is throw crashing the app.

This PR prevents `@react-native-cookies/cookies` from throwing such exception, preventing the app from crashing

# How to reproduce the crash

1. Be on a physical device
2. Open the Settings app and then navigate to All apps > Search for WebView > Disable

# How does the PR work

This PR postpone accessing the `CookieManager` instance (which could throw `MissingWebViewPackageException`) until when it's necessary, instead of doing when the native module is initialized.

### CODE REVIEW

File: android/src/main/java/com/reactnativecommunity/cookies/CookieManagerModule.java [hunk 1]
```diff
@@ -43,14 +43,21 @@
     private static final String CLEAR_BY_NAME_NOT_SUPPORTED = "Cannot remove a single cookie by name on Android";
     private static final String INVALID_DOMAINS = "Cookie URL host %s and domain %s mismatched. The cookie won't set correctly.";
 
-    private CookieManager mCookieManager;
     private CookieSyncManager mCookieSyncManager;
 
     CookieManagerModule(ReactApplicationContext context) {
         super(context);
         this.mCookieSyncManager = CookieSyncManager.createInstance(context);
-        this.mCookieManager = CookieManager.getInstance();
```
Comment (capezzbr) [inline android/src/main/java/com/reactnativecommunity/cookies/CookieManagerModule.java:52]: Lazy get `CookieManager` instance - basically try to only crash when actually using the library

**Comment by capezzbr** [comment]
cc @safaiyeh 🙇

**Comment by safaiyeh** [comment]
@capezzbr appreciate the PR! Let's get this deployed.

**Review by safaiyeh — APPROVED** [review]

**Pull request merged**